airflow/operators/__init__.py

```diff
@@ -88,7 +88,7 @@
     'jdbc_operator': ['JdbcOperator'],
     'mssql_operator': ['MsSqlOperator'],
     'mssql_to_hive': ['MsSqlToHiveTransfer'],
-    'slack_operator': ['SlackAPIOperator', 'SlackAPIPostOperator'],
+    'slack_operator': ['SlackAPIOperator', 'SlackAPIPostOperator', 'SlackPostOperator'],
```

Comment:
Don't know, that would depend on if anyone are using them. I guess the SlackAPIOperator may be used by custom operators to implement functionality different than Post. 

Perhaps marking them as deprecated? Didn't want to do that without discussion.